CRM/Member/Form/Membership.php

```diff
@@ -1685,7 +1679,7 @@ public function submit() {
         }
         if ($taxAmount) {
           $this->assign('totalTaxAmount', $totalTaxAmount);
-          $this->assign('taxTerm', CRM_Utils_Array::value('tax_term', $invoiceSettings));
+          $this->assign('taxTerm', $this->getSalesTaxTerm());
```

Comment:
Not sure why would need this on Submit.... unless it's being used when sending mails in which case this is the wrong place

Reply:
@eileenmcnaughton Perhaps add that as a code comment for now - everything in this PR is NFC as far as I can tell, but if we remove this line it may no longer be...